public/js/app/world-map.js

```diff
@@ -0,0 +1,189 @@
+    shadowPolygonSeries.mapPolygons.template.fillOpacity = 0.2;
+    shadowPolygonSeries.mapPolygons.template.strokeOpacity = 0;
+    shadowPolygonSeries.fillOpacity = 0.1;
+    shadowPolygonSeries.fill = am4core.color("#000");
+
+
+// Create map polygon series
+    var polygonSeries = chart.series.push(new am4maps.MapPolygonSeries());
+    polygonSeries.useGeodata = true;
+
+    polygonSeries.calculateVisualCenter = true;
+    polygonSeries.tooltip.background.fillOpacity = 0.2;
+    polygonSeries.tooltip.background.cornerRadius = 20;
+
+    var template = polygonSeries.mapPolygons.template;
+    template.nonScalingStroke = true;
+    template.fill = am4core.color("#f9e3ce");
+    template.stroke = am4core.color("#e2c9b0");
+
+    polygonSeries.calculateVisualCenter = true;
+    template.propertyFields.id = "id";
+    template.tooltipPosition = "fixed";
+    template.fillOpacity = 1;
+
+    template.events.on("over", function (event) {
+      if (event.target.dummyData) {
+        event.target.dummyData.isHover = true;
+      }
+    });
+    template.events.on("out", function (event) {
+      if (event.target.dummyData) {
+        event.target.dummyData.isHover = false;
+      }
+    });
+
+    var hs = polygonSeries.mapPolygons.template.states.create("hover");
+    hs.properties.fillOpacity = 1;
+    hs.properties.fill = am4core.color("#deb7ad");
+
+
+    var graticuleSeries = chart.series.push(new am4maps.GraticuleSeries());
+    graticuleSeries.mapLines.template.stroke = am4core.color("#fff");
+    graticuleSeries.fitExtent = false;
+    graticuleSeries.mapLines.template.strokeOpacity = 0.2;
+    graticuleSeries.mapLines.template.stroke = am4core.color("#fff");
+
+
+    var measelsSeries = chart.series.push(new am4maps.MapPolygonSeries());
+    measelsSeries.tooltip.background.fillOpacity = 0;
+    measelsSeries.tooltip.background.cornerRadius = 20;
+    measelsSeries.tooltip.autoTextColor = false;
+    measelsSeries.tooltip.label.fill = am4core.color("#000");
+    measelsSeries.tooltip.dy = -5;
+
+    var measelTemplate = measelsSeries.mapPolygons.template;
+    measelTemplate.fill = am4core.color("#bf7569");
+    measelTemplate.strokeOpacity = 0;
+    measelTemplate.fillOpacity = 0.75;
+    measelTemplate.tooltipPosition = "fixed";
+
+
+    var hs2 = measelsSeries.mapPolygons.template.states.create("hover");
+    hs2.properties.fillOpacity = 1;
+    hs2.properties.fill = am4core.color("#86240c");
+
+    polygonSeries.events.on("inited", function () {
+      polygonSeries.mapPolygons.each(function (mapPolygon) {
+        var count = data[mapPolygon.id];
+
+        if (count > 0) {
+          var polygon = measelsSeries.mapPolygons.create();
+          polygon.multiPolygon = am4maps.getCircle(mapPolygon.visualLongitude, mapPolygon.visualLatitude, Math.max(0.2, Math.log(count) * Math.LN10 / 10));
+          polygon.tooltipText = mapPolygon.dataItem.dataContext.name + ": " + count;
```

Comment:
String format? I remember sth in my head about js not being capable of thattthough. Doesn't warrent a helper funct.